src/definitions/modules/modal.js

```diff
@@ -91,6 +92,39 @@ $.fn.modal = function(parameters) {
       module  = {
 
         initialize: function() {
+          if(!$module.hasClass('modal')) {
+            module.create.modal();
+            settings.onHidden = function(){
```

Comment:
We should add an option to define what we want to do when closing the model (`hide` by default, or `remove`).

Reply:
O, but i would use `remove` as default, because such modals are basically always created temporary. If we hide them only as default they will be kept in the domtree, because each call to $('body').modal() will create a complete new DOM node.
Imagine 200 "alert" modals in  an app session. It will blew up the DOM tree makes the app slower and slower and, unless the user references to the temporary modal afterwards, it will never be referenced anymore.

Btw, i just copied over your logic from fui-alert, where you also remove the modal once closed 😉

Reply:
It was a comment for the whole modal system, including the old style, but yeah, you're right, if we implements this, `remove` should be the default. Let's postpone this until further.

Reply:
I will change it that way, so the `onHidden` function will only be generated (and removing the modal) when there isn't already a `onHidden` function set, so if anybody wants to keep the modal DOM, then thoese people can simply provide their own `onHidden` function which will do whatever they need.
For that, i will make the default of `onHidden` false and check for their existance before actually calling it. Agreed?

Reply:
Yup ! That'll do the trick 👍

Reply:
Implemented by fc073b4b6d8a1f1067889d259bfcbc8566182757
Please review again

Reply:
I also agree to use `remove` as default. I've been creating dynamic modals in my project mostly for Ajax content and they are removed from DOM tree after closing. Otherwise, there'll be the duplicate modals when calling again to render and they'll conflict with the ID using by the same modal.